enos/ansible/roles/patches/files/mariadb_start.yml

```diff
@@ -0,0 +1,32 @@
+---
+- name: Starting mariadb container
+  docker_container:
+    image: "mariadb"
+    name: "mariadb"
+    volumes:
+      - "{{ node_config_directory }}/mariadb/:/etc/mysql/conf.d:ro"
+      - "/etc/localtime:/etc/localtime:ro"
+      - "mariadb:/var/lib/mysql"
+      - "kolla_logs:/var/log/kolla/"
+    network_mode: host
+    env:
+      MYSQL_ROOT_PASSWORD: "{{ database_password }}"
+  when: delegate_host != 'None' or
+        ( groups['mariadb'] | length ) == 1 or
+        ( delegate_host == 'None' and mariadb_recover_inventory_name is not defined and inventory_hostname != groups['mariadb'][0] ) or
+        ( delegate_host == 'None' and mariadb_recover_inventory_name is defined and inventory_hostname != mariadb_recover_inventory_name )
+
+
+# TODO(jeffrey4l), remove the task check when the wair_for bug is fixed
+# https://github.com/ansible/ansible-modules-core/issues/2788
+- name: Waiting for MariaDB service to be ready
+  wait_for:
+    host: "{{ hostvars[inventory_hostname]['ansible_' + api_interface]['ipv4']['address'] }}"
```

Comment:
should this be `{{ api_interface_address }}` (maybe kolla related, I don't know the original file)?